po/com.vysp3r.ProtonPlus.pot

```diff
@@ -56,50 +61,47 @@ msgstr ""
 msgid "Install the runner"
 msgstr ""
 
-#: src/widgets/info-box.vala:261 src/widgets/info-box.vala:273
-#: src/widgets/info-box.vala:287 src/launchers/steam.vala:587
-#: src/launchers/steam.vala:612 src/launchers/steam.vala:614
-msgid "The installation of "
-msgstr ""
-
 #: src/widgets/info-box.vala:261 src/launchers/steam.vala:614
-msgid " was cancelled"
+#, c-format
+msgid "The installation of %s was cancelled."
 msgstr ""
 
-#: src/widgets/info-box.vala:273 src/launchers/steam.vala:553
-#: src/launchers/steam.vala:587
-msgid " was started"
+#: src/widgets/info-box.vala:273 src/launchers/steam.vala:587
+#, c-format
+msgid "The installation of %s has begun."
```

Comment:
We should not put any %s in the texts otherwise this defy the purpose of all the modifications we made regarding that.

Reply:
We should keep it the way it was before your modification

Reply:
We have to do this change. It's the only proper way to translate strings in applications, to avoid the problems that are mentioned in the description of the related commit:

https://github.com/Vysp3r/ProtonPlus/pull/211/commits/1141d59050b52abb5fdf35f044cca1b452b6c3f4

There are no problems whatsoever with this change. It finally makes the application easy to translate. Languages can now say "Proton-GE esta prueba blabla habla habla." for spanish for example, whereas something like Russian could do "Dobro dobro lalal da da da Proton-Ge."

It finally gives clarity and freedom to the translators to make natural text for the reader.

The `%s` means: This is where the special word (subject matter of the sentence, such as the runner title) will be inserted.

It's perfect. :)

Reply:
Thanks to this change I will finally have the energy to contribute a Swedish translation. The old fragments with important whitespace like ` is done.` and `The process has begun for ` etc was so headache-inducing that I didn't want to contribute any translation.

Reply:
It does make sense, I will just have to be more cautious about translations since I do not want someone removing the said %s.

Reply:
I don't think that's a worry. If I remember right, the weblate or whatever the website is called will validate that the translation contains `%s` and other signifiers at the exact same amount as the original string.

And `%s` is a lot easier for the translator than to remember "you are NOT ALLOWED to remove the space after/before the sentence fragment you are translating" (the old system hehe). :D

Reply:
@Vysp3r Btw, the techniques used for translating are visible in the linked commit, but here they are for a quick reference:

1. When you want to make long, multi-paragraph string with multiple separate sentences, create a `"%s\n\n%s"` style template and then printf each individual translated sentence into it, like this: `"%s\n\n%s".printf (_("It looks like you currently have another version of STL which was not installed by ProtonPlus."), _("Do you want to delete it and install STL with ProtonPlus?")` so that the translator does not have to care about newlines between the strings.
2. When you want to do a single sentence with a subject matter, use this pattern instead: `_("The deletion of %s is complete.").printf (title)`
3. If you need a multi-paragraph string (like the first bullet point) but you need dynamic subject words in some of the sentences, you can simply combine both techniques, by putting `_().printf()` on the specific sentences that contain "%s".